### PR TITLE
[v1.10.0] [zos_copy] Enhance zos_copy performance when copying multiple PDS members 

### LINKED ISSUE #764: [Enhancement] [zos_copy] Improve performance when copying an entire PDS with thousands of members
### Is there an existing issue for this?

- [X] There are no existing issues.

### Ansible module

zos_copy

### Enhancement or feature description

looks like zos_copy is copying 1 member at a time

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Fixes #764 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

zos_copy

### CODE REVIEW

**Comment by fernandofloresg** [comment]
Screenshot 2024-01-29 at 9 36 36 AM

Screenshot 2024-01-29 at 9 39 22 AM

File: plugins/modules/zos_copy.py [hunk 1]
```diff
@@ -1705,33 +1707,56 @@ def copy_to_pdse(
...
                 overwritten_members.append(destination_member)
             else:
                 new_members.append(destination_member)
-
+            bulk_src_members += "{0} ".format(src_member)
+
+        # Copy section
+        if src_ds_type == "USS" or self.asa_text or len(src_members) == 1:
+            """
+            USS -> MVS : Was kept on member by member basis bc file names longer that 8
```
Comment (richp405) [inline plugins/modules/zos_copy.py:1723]: Minor: should be 'than' not 'that'

**Review by richp405 — APPROVED** [review]
I mentioned a minor change (spelling) on zos_copy.py comments.  Not a show stopper.

**Review by rexemin — APPROVED** [review]
I still can't believe that you're able to bulk copy members of a data set in this way

File: plugins/modules/zos_copy.py (hunk 1 above)
Comment (fernandofloresg) [reply plugins/modules/zos_copy.py]: hey thanks rich, did updated it and run the ansible certification tests to validate.

Screenshot 2024-01-29 at 7 34 25 PM

**Pull request merged**